syntax="proto3";

package map_renderer_proto;

message Rgb {
	uint32 red = 1;
	uint32 green = 2;
	uint32 blue = 3;
}

message Rgba {
	uint32 red = 1;
	uint32 green = 2;
	uint32 blue = 3;
	double opacity = 4;
}

message Color {
	oneof inx {
        string text = 2;
        Rgb rgb = 3;
        Rgba rgba = 4;
    } 
}